openfl/interface/collaborator.py

```diff
@@ -77,7 +78,27 @@ def start_(plan, collaborator_name, data_config):
 
     # TODO: Need to restructure data loader config file loader
     logger.info(f"Data paths: {plan_obj.cols_data_paths}")
-    echo(f"Data = {plan_obj.cols_data_paths}")
+    # this check is added to avoid mock objects failing
+    if isinstance(plan_obj, Plan):
+        if "network" in plan_obj.config and "settings" in plan_obj.config["network"]:
+            if plan_obj.config["network"]["settings"].get("enable_remote_attestation", False):
```

Comment:
same here

Reply:
This check is added as unit tests are sending Mock Object which doesn't have config or is an instance of PLAN. Hence added the check.